libs/domain/cart/multi/providers.ts

```diff
@@ -0,0 +1,7 @@
+import { Provider } from '@spryker-oryx/di';
+import { provideExperienceData } from '@spryker-oryx/experience';
+import { cartsPage } from './presets';
+
+export const multiCartsProviders: Provider[] = [
+  provideExperienceData(cartsPage),
+];
```

Comment:
I believe we started this wrong, me actually. Would be better to export the data from the domain, but not provide it by default. That's what we still do in presets.